students/yunhyung-lee/project_westagram/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class User(models.Model):
+    email       = models.CharField(max_length=100)
+    password    = models.CharField(max_length=30)
+    phone_num   = models.CharField(max_length=30)
+    user_name   = models.CharField(max_length=30)
```

Comment:
- 변수명은 확실히 해주세요.
- 이미 user 정보를 담고 있기 때문에 name로만 표현 하셔도 됩니다.
```suggestion
    phone_number   = models.CharField(max_length=30)
    name           = models.CharField(max_length=30)
```

Reply:
넵 수정했습니다 다시 푸시하겠습니다